src/engraving/dom/chord.cpp

```diff
@@ -3219,7 +3219,7 @@ void Chord::computeKerningExceptions()
     }
 }
 
-Ornament* Chord::findOrnament() const
+Ornament* Chord::findOrnament(bool forPlayback) const
```

Comment:
Could you rename this flag? Something like "includingEndingSpanners" (or something else)

Reply:
My intention was that the name indicates the purpose for calling the function, i.e. whether you're doing it for layout or playback purposes - that currently this means checking "ending spanners" when determining playback is an implementation detail that might change in future.

Reply:
Functions should never know by whom or how they will be called